### PR TITLE
Add feature-index (FIX) fixture data

### LINKED ISSUE #3911: Add feature-index (FIX) fixture data
**Is your feature request related to a problem? Please describe.**
We want to POC feature-index for world service in Simorgh. To start this process we should add some fixture data for a few services.

**Describe the solution you'd like**
Add fixture data for the following FIX assets to Simorgh.

https://ares-broker.api.bbci.co.uk/api/index/afrique/48465371
https://ares-broker.api.bbci.co.uk/api/index/hindi/39414759
https://ares-broker.api.bbci.co.uk/api/index/marathi/43599597
https://ares-broker.api.bbci.co.uk/api/index/persian/39120696

For now add the fixture data to a new dir called `fix` under `/data/<service>/fix`

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by AlistairGempf**
Resolves #3911

**Overall change:** Add fixture data for feature index.

**Code changes:**

- Add fixture data for feature index.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/README.md [hunk 1]
```diff
@@ -1,50 +0,0 @@
-# Cypress tests and our approach to E2E testing
```
Comment (dr3) [inline cypress/README.md]: Howcome this is being deleted?

File: data/afrique/fix/index.json [hunk 2]
```diff
@@ -0,0 +1,3398 @@
+{
+  "metadata": {
+    "id": "urn:bbc:ares::index:afrique/48465371/desktop/domestic",
+    "locators": {
+      "assetUri": "/afrique/48465371",
```
Comment (dr3) [inline data/afrique/fix/index.json]: Should this file not be called `48465371.json`? A service can have multiple, unlike frontpages which has 1, so `index` is fine.

**Review by dr3 — DISMISSED** [review]

File: cypress/README.md (hunk 1 above)
Comment (AlistairGempf) [reply cypress/README.md]: I have no idea how that happened

File: data/afrique/fix/index.json (hunk 2 above)
Comment (AlistairGempf) [reply data/afrique/fix/index.json]: Ah ok, that makes sense. Thanks!

**Review by j-pendlebury — APPROVED** [review]
Looks good to me 👍

**Comment by ghost** [comment]
Rename `fix` dir to `feature_index` to provide clarity on the purpose of the folder.

**Comment by DenisHdz** [comment]
> Rename `fix` dir to `feature_index` to provide clarity on the purpose of the folder.

I would say `featureIndex`. In our naming conventions:
**Files and directories should be camelCase, e.g. `storyBody.jsx`**

**Review by ghost — APPROVED** [review]

**Pull request merged**